app/assets/stylesheets/components/_card.scss

```diff
@@ -0,0 +1,27 @@
+.card {
+  max-width: 350px;
+  box-shadow: 0 0 5px rgba($color: #000000, $alpha: 0.3);
+  border-radius: 7px;
+
+  img {
+    width: 100%;
+    height: 165px;
+    object-fit: cover;
+    object-position: center;
+    border-radius: 3px;
+    padding-bottom: 7px;
+  }
+
+  i {
+    color: $primary;
+    margin-top: 9px;
+  }
+
+  p {
+    margin: 0;
+  }
+}
+
+.card-container {
+  padding: 15px !important;
```

Comment:
do we really need to use the `!important` here?

Reply:
Fixed!